homeassistant/components/history/__init__.py

```diff
@@ -110,6 +109,15 @@ async def async_setup(hass, config):
     return True
 
 
+class LazyState(history_models.LazyState):
+    """A lazy version of core State."""
+
+    @deprecated_function("homeassistant.components.recorder.models.LazyState")
```

Comment:
We shouldn't decorate the init method, I think. We don't want the class to be used so we should decorate that.